app/workers/unmerge_worker.rb

```diff
@@ -6,16 +6,40 @@ class UnmergeWorker
 
   sidekiq_options queue: 'pull'
 
-  def perform(from_account_id, into_account_id)
+  def perform(from_account_id, into_id, type = 'home')
```

Comment:
See `MergeWorker` above: I think `'home'` should be a symbol and think that maybe keyword arguments might be a nice touch.